lib/internal/worker.js

```diff
@@ -152,7 +152,7 @@ class Worker extends EventEmitter {
                                    env === process.env ? null : env,
                                    options.execArgv,
                                    parseResourceLimits(options.resourceLimits),
-                                   !!options.trackUnmanagedFds);
+                                   !!(options.trackUnmanagedFds ?? true));
```

Comment:
If this needs a backport should this be `||`

Reply:
I think this is semver-major anyway :)